modules/ossm-exposing-service-using-istio-gateway-and-virtualservice.adoc

```diff
@@ -0,0 +1,133 @@
+:_mod-docs-content-type: PROCEDURE
+[id="ossm-exposing-service-using-istio-gateway-and-virtualservice_{context}"]
+= Exposing a service using an Istio gateway and a VirtualService
+:context: ossm-exposing-service-using-istio-gateway-and-virtualservice
+
+This procedure configures a gateway that was deployed using gateway injection to expose a service in the mesh.
```

Comment:
Adding some info here reinforcing that the gateway is configured using the Istio `Gateway` and `VirtualService` resources, and also making a distinction between exposing a service to traffic outside the mesh and exposing the gateway to traffic outside the cluster.

```suggestion
This procedure uses the Istio `Gateway` and `VirtualService` resources to configure a gateway that was deployed using gateway injection. The gateway is configured to expose a service within the mesh to traffic outside the mesh. After being configured, the gateway is exposed to traffic outside the cluster by setting the gateway `Service` to type `LoadBalancer`.
```

Reply:
chatted and updated.